package.json

```
@@ -56,6 +56,11 @@
       "type": "object",
       "title": "sway-lsp",
       "properties": {
+        "sway-lsp.path": {
```

Let's make this more specific, in case there are other paths we want to add in the future.

```suggestion
        "sway-lsp.lsp-bin-path": {
```